component/avatar.css

```
@@ -0,0 +1,27 @@
+.avatar-xl {
+  width: 64px;
+  border-radius: 50%;
+  vertical-align: middle;
+}
+
+.avatar-lg {
+  width: 48px;
+  border-radius: 50%;
+  vertical-align: middle;
+}
+
+.avatar-default {
+  width: 32px;
+  border-radius: 50%;
+  vertical-align: middle;
+}
+.avatar-sm {
+  width: 24px;
+  border-radius: 50%;
+  vertical-align: middle;
+}
+.avatar-xs {
+  width: 16px;
+  border-radius: 50%;
+  vertical-align: middle;
+}
```

```suggestion
.avatar-xl, .avatar-sm,.avatar-xs,.avatar-default,.avatar-lg {
  border-radius: 50%;
  vertical-align: middle;
}

```
DRY code. You can set the width individually , but club other common properties as above

---

Will do that.